CONTRIBUTING.md

```diff
@@ -1,100 +1,149 @@
-1. Entries will be listed in chronological order automatically.
+  **7.** Make the desired changes.
 
-#### <a name="events">Adding an Event</a> [create new issue](https://github.com/layer5io/layer5/issues/new?assignees=&labels=area%2Fevents&template=events.md&title=%5BEvent%5D)
+  **8.** Run the site locally to preview changes.
 
+  ```
+  make site
+  ```
 
-## Contributor Instructions
+This will run a local web server with "live reload" conveniently enabled. ( **NOTE**: while using make command on Windows, there sometimes arises an error in identifying the command even after it is installed (unrecognized command), this is because the PATH for the binary might not be set correctly ).
 
-Before building the site, ensure that you have the following dependencies installed on your system.
+  **9.** Track your changes.
 
-- [Nodejs and npm](https://nodejs.org/en/): To verify run `node -v` & `npm -v`
+  ```
+  git add .
+  ```
 
-### Building the site
+  **10.** Commit your changes. To contribute to this project, you must agree to the [Developer Certificate of Origin (DCO)](## Signing-off on Commits (Developer Certificate of Origin)) for each commit you make.
 
-Run the following:
+  ```
+  git commit --signoff -m "<commit subject>"
+  ```
 
-1. `make setup-libs`-To install the dependencies for the running the site.
-1. `make site`-To run the website locally.
 
-Please contribute! The Layer5 site uses Gatsby. The process of contributing to documentation follows this flow:
+  **11.** Push the committed changes in your feature branch to your remote repo.
 
-1. Fork a copy of this repo.
-1. Get a local clone of your fork of the site.
-`git clone https://github.com/layer5io/layer5`
-1. Switch to the `master` branch.
-`git switch master`
-1. Create and checkout a new branch to make changes within
-`git checkout -b <my-changes-branchname>`
-1. Edit site/add content.
-`vi <specific page>.md` # or use your favorite IDE
-1. Run site locally to preview changes.
-`make site` # this will run a local web server with "live reload" conveniently enabled.
-(NOTE: while using `make` command on windows, there sometimes arises an error in identifying the command even after it is installed(unrecognized command), this is because the PATH for the binary might not be set correctly.)
-1. Commit your changes to your remote branch.
-`git commit --signoff -m"<commit subject>`
-1. Push your changes
-`git push origin <my-changes-branchname>`
-1. Open a [pull request](../../pulls) (in your web browser) against the `master` branch on https://github.com/layer5io/layer5.
+  ```
+  git push -u origin <your_branch_name>
+  ```
 
-### <a name="commit-signing">Signing-off on Commits (Developer Certificate of Origin)</a>
+  **12.** To create a pull request, click on `compare and pull requests`. Please ensure that you compare your feature branch to the desired branch of the repo you are supposed to make a PR to.
```

Comment:
```suggestion
  **12.** Once you’ve committed and pushed all of your changes to GitHub, go to the page for your fork on GitHub, select your development branch, and click the pull request button.. Please ensure that you compare your feature branch to the desired branch of the repo you are supposed to make a PR to.``` If you need to make any adjustments to your pull request, just push the updates to GitHub. Your pull request will automatically track the changes in your development branch and update it.

Reply:
Nice point!

Reply:
done!